torch/functional.py

```diff
@@ -318,6 +408,120 @@ def meshgrid(*tensors, **kwargs):
     return torch._C._VariableFunctions.meshgrid(tensors)
 
 
+def multi_mm(*matrices):
+    r"""Returns the matrix product of the :math:`N` 2-D tensors. This product is efficiently computed
+    using the matrix chain order algorithm which selects the order in which incurs the lowest cost in terms
+    of arithmetic operations (`[CLRS]`_). Note that :math:`N` needs to be greater than or equal to 2; if equal to 2
+    then a trivial matrix-matrix product is returned.
```

Comment:
Well the case of one matrix can be handled trivially as well. For zero we could return the identity matrix, but we don't know the sizes 😕